src/Protocol/Diaspora.php

```diff
@@ -3540,12 +3569,12 @@ private static function constructLike(array $item, array $owner)
 			$positive = "false";
 		}
 
-		return(["author" => self::myHandle($owner),
+		return ["author" => self::myHandle($owner),
 				"guid" => $item["guid"],
 				"parent_guid" => $parent["guid"],
 				"parent_type" => $target_type,
 				"positive" => $positive,
```

Comment:
Please re-indent this array and convert quotes.
```suggestion
		return [
			'author' => self::myHandle($owner),
			'guid' => $item['guid'],
			'parent_guid' => $parent['guid'],
			'parent_type' => $target_type,
			'positive' => $positive,
			'author_signature' => '',
		];
```